django/test/runner.py

```diff
@@ -97,25 +97,47 @@ def debug(self, error):
         pdb.post_mortem(traceback)
 
 
-class RemoteTestResult:
+class DummyList:
     """
-    Record information about which tests have succeeded and which have failed.
+    Dummy list class for faking storage of results in unittest.TestResult. We
+    do this because RemoteTestResult replaces the way of recording.
+    """
+    __slots__ = []
 
-    The sole purpose of this class is to record events in the child processes
-    so they can be replayed in the master process. As a consequence it doesn't
-    inherit unittest.TestResult and doesn't attempt to implement all its API.
+    def append(self, item):
+        pass
 
-    The implementation matches the unpythonic coding style of unittest2.
+
+class RemoteTestResult(unittest.TestResult):
+    """
+    Extend unittest.TestResult to record events in the child processes so they
+    can be replayed in the master process. Events include things like which
```

Comment:
How about using "parent" here instead of "master"? Seems like a better fit with "child processes".

Reply:
Yes and I remember the master/slave controversy.